core/src/main/java/io/questdb/cairo/vm/api/MemoryR.java

```diff
@@ -67,21 +69,44 @@
 
     void getLong256(long offset, CharSink sink);
 
+    default void getLong256(long offset, Long256Acceptor sink) {
+        long addr = addressOf(offset + Long.BYTES * 4);
+        sink.setAll(
+                Unsafe.getUnsafe().getLong(addr - Long.BYTES * 4),
+                Unsafe.getUnsafe().getLong(addr - Long.BYTES * 3),
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The result of `Integer` multiplication is cast into a larger type. If you think the result of the multiplication could exceed the size of an `Integer`, consider casting the arguments of the multiplication to avoid arithmetic overflows. For more information, see [CWE-190](https://cwe.mitre.org/data/definitions/190.html).